app/lib/frontend/handlers/custom_api.dart

```diff
@@ -251,6 +251,7 @@ Future<VersionScore> packageVersionScoreHandler(
       maxPoints: card?.maxPubPoints,
       likeCount: pkg.likes,
       popularityScore: card?.popularityScore,
+      tags: card?.derivedTags,
```

Comment:
Can we exposed both `derivedTags` and `assignedTags`?

Reply:
Ah, good point, this doesn't contain the tags coming from the `Package` or `PackageVersion`... will update.